managed/CounterStrikeSharp.API/Core/GameData.cs

```diff
@@ -31,33 +32,45 @@ public class Offsets
     [JsonPropertyName("linux")] public int Linux { get; set; }
 }
 
-public static class GameData
```

Comment:
are you sure that `GameData` should be retrieved as a service instead of `static` access?

Reply:
GameData is still a static class, GameDataProvider is the service used from core to load the game data. Ideally I would prefer if everything pulled configs from the container (even plugin configs), because it is just better in every way than statics (i.e. we have context of which plugin called it through container etc.)

Reply:
I agree on DI overall, just making sure we want to use services for gamedata and coreconfig (also other things that are not mentioned if there is any)